docs: add "Link a Server to an Application" post (#704)
devopness/devopness: docs/applications/link-server-to-application.md

---
title: Link a Server to an Application
intro: Link a server to an application to select which servers an application can be deployed.
links:
    overview:
    quickstart:
    previous:
        - /docs/configuration-files/add-configuration-file
    next:
        - /docs/applications/deploy-application
    guides:
    related:
    featured:
---

1. On Devopness, navigate to a project then select an environment
1. Find the `Applications` card
1. Click `View` in the `Applications` card to see a list of existing `Applications`
1. Click `DETAILS` on the application you want to link a server
1. On the upper-right corner of the application details view, click `...`
1. Use the drop-down menu to choose `Linked resources`
1. On the upper-right corner of the `Linked resources` list, click `LINK TO A SERVER`
1. Follow the prompts then click `Link`

> 1. In the `Linked Resource` list, the selected server will be listed in the `DEPENDS ON` table

Author: ```suggestion
1. In the `Linked Resource` list, find the selected server from the previous step in the `DEPENDS ON` table
```

>     - The selected server will be listed in the `DEPENDS ON` table on the `Linked resources` list

Reviewer: `Follow the prompts ...` + `the selected ...` is really the pattern we've been following in other topics that use select/drop down elements? 🤔 
`Follow the prompts` was usually adopted (by you) when there were more than one thing/element to interact with.

Please re-consider which approach is the one you recommend and let's stick to a single one.

Author: I agree, it's a pattern change.

Added a new step to the list, could you please re-review?

Author: Applied the same pattern used in `docs/cloud-provider-credentials/add-aws-credential.md`

```md
...
1. Follow the prompts then click `Create user`
1. In the `Users` list, click the username link of the user recently created
...
```

Author: @jfoliveira could you review if the applied changes solves the thread?

> 1. In the `Linked Resource` list, find the selected server from the previous step in the `DEPENDS ON` table

Reviewer: `find the selected server`? Why? For what? 

If the intention is to have a closing statement, it should not be a "step/order" to the reader, should just be an informational note
```
> In the <element>, the <info> can be seen in <info location>
> In the <element>, the <info> will be displayed in <info location>
etc
```

If it's an imperative instruction, it needs to guide the user o do something with the info that was found
```
find <info> to <...>
```

Author: Post updated

> 1. In the `Linked Resource` list, find the selected server from the previous step in the `DEPENDS ON` table

Author: ```suggestion
1. In the `Linked Resource` list, the recently linked server can be seen in the `DEPENDS ON` table
```
